plugins/inventory/proxmox.py

```diff
@@ -234,13 +241,22 @@ def _get_agent_network_interfaces(self, node, vmid, vmtype):
                 )
             )['result']
 
+            if "error" in ifaces:
+                if "class" in ifaces["error"]:
+                    # This happens on Windows, even though qemu agent is running, the IP address
+                    # cannot be fetched, as it's unsupported, also a command disabled can happen.
+                    errorClass = ifaces["error"]["class"]
+                    if errorClass in ["Unsupported"]:
+                        self.display.v("Retrieving network interfaces from guest agents on windows with older qemu-guest-agents is not supported")
+                    elif errorClass in ["CommandDisabled"]:
+                        self.display.v("Retrieving network interfaces from guest agents has been disabled")
+                return result
+
             for iface in ifaces:
                 result.append({
                     'name': iface['name'],
-                    'mac-address': iface['hardware-address'],
-                    'ip-addresses': [
-                        "%s/%s" % (ip['ip-address'], ip['prefix']) for ip in iface['ip-addresses']
-                    ]
+                    'mac-address': iface['hardware-address'] if 'hardware-address' in iface else "00:00:00:00:00:00",
```

Comment:
I looked into 00:00:00:00:00:00 a bit and it seems preferable to just default to `""`.
You would only see that MAC Address in ARP requests, but 00:00:00 is a valid OUI so could be misleading.

Reply:
I'm fine with that, if that's what is needed. The only reason it's there is as all loopback and local interfaces default to mac of 00:00......
As long as it's defined, as there are cases where hardware-address is not defined and the playbook crashed. 

Should I just change it to empty string or None?

Reply:
Having a falsy value (`''` or `None`) makes it easier to distinguish between a real MAC and this case.

Also not providing it makes it possible to use `| default(...)` in jinja2.

Reply:
I guess `''` is the better value then. `None` is more for Python programmers and less for Ansible users IMO :)

Reply:
Changed it to a blank string. Let me know if this is good.